tests/app/main/test_briefs.py

```diff
@@ -2086,3 +2092,203 @@ def test_evaluation_methods_shown_with_a_or_an(self, data_api_client):
+            '**Essential two with markdown**'
+        ]
+        self.brief['briefs']['niceToHaveRequirements'] = [
+            '<h1>n2h one with xss</h1>',
+            '**n2h two with markdown**'
+        ]
+        self.brief_responses['briefResponses'][0]['niceToHaveRequirements'] = [
+            {
+                'yesNo': True,
+                'evidence': 'Did a thing'
+            },
+            {
+                'yesNo': False
+            }
+        ]
+
+        data_api_client.get_brief.return_value = self.brief
+        data_api_client.get_framework.return_value = self.framework
+        data_api_client.is_supplier_eligible_for_brief.return_value = True
+        data_api_client.find_brief_responses.return_value = self.brief_responses
+
+        res = self.client.get('/suppliers/opportunities/1234/responses/result')
+        data = res.get_data(as_text=True)
+        doc = html.fromstring(data)
+
+        assert "&lt;h1&gt;Essential one with xss&lt;/h1&gt;" in data
+        assert "&lt;h1&gt;n2h one with xss&lt;/h1&gt;" in data
+        assert len(doc.xpath('//h1')) == 1
+
+        assert "**Essential two with markdown**" in data
+        assert "<strong>Essential two with markdown</strong>" not in data
+        assert "**n2h two with markdown**" in data
+        assert "<strong>n2h two with markdown</strong>" not in data
+
+    def test_view_response_result_submitted_ok(self, data_api_client):
+        data_api_client.get_brief.return_value = self.brief
+        data_api_client.get_framework.return_value = self.framework
+        data_api_client.is_supplier_eligible_for_brief.return_value = True
+        data_api_client.find_brief_responses.return_value = self.brief_responses
+        res = self.client.get('/suppliers/opportunities/1234/responses/result')
+
+        assert res.status_code == 200
+        doc = html.fromstring(res.get_data(as_text=True))
+        assert doc.xpath('//p[contains(@class, "banner-message")]')[0].text.strip() == \
+            "Your application for ‘I need a thing to do a thing’ has been submitted."
+
+    def test_view_response_result_submitted_ok_for_closed_brief(self, data_api_client):
+        closed_brief = self.brief.copy()
+        closed_brief['briefs']['status'] = 'closed'
+        data_api_client.get_brief.return_value = closed_brief
+        data_api_client.get_framework.return_value = self.framework
+        data_api_client.is_supplier_eligible_for_brief.return_value = True
+        data_api_client.find_brief_responses.return_value = self.brief_responses
+        res = self.client.get('/suppliers/opportunities/1234/responses/result')
+
+        assert res.status_code == 200
+        doc = html.fromstring(res.get_data(as_text=True))
+        assert doc.xpath('//p[contains(@class, "banner-message")]')[0].text.strip() == \
+            "Your application for ‘I need a thing to do a thing’ has been submitted."
+
+    def test_essential_skills_shown_with_response(self, data_api_client):
```

Comment:
Agreed. I'll add some tests to see if the actual data is coming through and, that working well, it'll make sense to combine both of those.